hack/get_image_info.sh

```diff
@@ -3,16 +3,15 @@
 image=$1
 version=$2
 
-kube_image=kubebuilder/kube-rbac-proxy
-kube_image_version=v0.8.0
+docker pull ${image}:${version}
```

Comment:
I think this will pull the image for the local architecture and not the manifest?
It should be just a matter of changing the Accept header to make the current script work

Reply:
I tried changing the request header, and that works for getting the repo digest, but not for getting the info for the created at time 😞 The script is getting the repo digest not the actual image digest, so yes this is pulling an image for the current arch, but the `RepoDigests` remains consistent between all images.  I can switch it to curl for the repo digest and use the image pull for just fetching the createdAt time, but I don't see the point in doing both 😅Happy to switch to a different solution if you can find one for getting the timestamp!

Reply:
I've opened #85 🙂